.github/workflows/test.yml

```diff
@@ -76,6 +76,12 @@ jobs:
           sudo apt-get update && sudo apt-get install -y python3-venv
           sudo --preserve-env=MONGODB_TEST_OUTPUT_URI,GITHUB_SHA,GITHUB_RUN_ID,GITHUB_RUN_NUMBER,GITHUB_RUN_ATTEMPT,GITHUB_X_PR_BASE_SHA,GITHUB_X_PR_BASE_REF,GITHUB_X_HEAD_REF,GITHUB_X_HEAD_SHA,GITHUB_REPOSITORY -Hu testbot bash -lc 'make test-go'
 
+      - name: Run CLI linter tests
```

Comment:
Maybe lint before running unit tests? 

Unit tests take forever, and linting should be fast